lib/AArch64Parser.mly

```diff
@@ -180,6 +183,12 @@ cond:
 | LE { A.LE }
 | LT { A.LT }
 
+label_addr:
+| HEX LABEL { Printf.sprintf "0x%x" $1 }
+| HEX       { Printf.sprintf "0x%x" $1 }
+| NUM LABEL { Printf.sprintf "%d" $1 }
+| NUM       { Printf.sprintf "%d" $1 }
+| NAME      { $1 }
```

Comment:
Why introducing numerical labels? Cannot TBNZ and TBZ have a NAME last argument as other branch instructions do?